src/components/select/select.tsx

```diff
@@ -4,24 +4,38 @@ import { DropdownIcon } from '../icons';
 
 import styles from './select.module.scss';
 
-export interface SelectProps {
-    value: string;
-    onValueChange: (value: string) => void;
+export interface SelectProps<V extends string> {
+    value: V;
+    onValueChange: (value: V) => void;
     placeholder?: string;
     children: React.ReactNode;
+    className?: string;
+    dropdownClassName?: string;
+    customSelectedValue?: React.ReactNode;
```

Comment:
I don't understand the difference between this and placeholder.

Reply:
Oh, is it like `renderValue`?

Reply:
Yes

Reply:
Updated